chainer_chemistry/datasets/tox21.py

```diff
@@ -107,20 +107,25 @@ def postprocess_label(label_list):
 
 
 def _get_tox21_filepath(dataset_type):
-    """Returns a filepath in which the tox21 dataset is cached.
+    """Returns a file path in which the tox21 dataset is cached.
 
-    Thie function returns a filepath in which `dataset_type`
+    This function returns a file path in which `dataset_type`
     of the tox21 dataset is cached.
-    Not that this function does not check if the dataset actually
-    has been downloaded or not.
+    Note that this function does not check if the dataset has actually
+    been downloaded or not.
 
     Args:
         dataset_type(str): Name of the target dataset type.
             Either 'train', 'val', or 'test'.
 
-    Returns (str): filepath for the tox21 dataset
+    Returns (str): file path for the tox21 dataset
 
     """
+    valid_dataset_types = {'train', 'val', 'test'}
+    if not dataset_type in valid_dataset_types:
```

Comment:
Could you use already defined constant value `_config.keys()` instead of defining again here?